elements/pl-string-input/pl-string-input.py

```diff
@@ -27,16 +27,21 @@ def render(element_html, data):
     label = pl.get_string_attrib(element, 'label', None)
     suffix = pl.get_string_attrib(element, 'suffix', None)
     display = pl.get_string_attrib(element, 'display', 'inline')
-    remove_leading_trailing = pl.get_string_attrib(element, 'remove-leading-trailing', False)
-    remove_spaces = pl.get_string_attrib(element, 'remove-spaces', False)
+    remove_leading_trailing = pl.get_boolean_attrib(element, 'remove-leading-trailing', True)
+    remove_spaces = pl.get_boolean_attrib(element, 'remove-spaces', False)
     placeholder = pl.get_string_attrib(element, 'placeholder', None)
 
     if data['panel'] == 'question':
         editable = data['editable']
         raw_submitted_answer = data['raw_submitted_answers'].get(name, None)
 
+        if remove_leading_trailing:
+            trailing_hint = 'Leading and trailing spaces will be removed from your answer.'
+        else:
+            trailing_hint = 'Leading and trailing spaces will be allowed (included as part of your answer).'
```

Comment:
Perhaps: `will be left as part of your answer.`

Reply:
done